playbooks/spark-aarch64-build/run.yaml

```diff
@@ -0,0 +1,84 @@
+- hosts: all
+  become: yes
+  tasks:
+    - name: Build spark
+      shell:
+        cmd: |
+          set -ex
+          apt-get update -y
+
+          # Install java
```

Comment:
Please use role `install-openjdk` and `install-maven`

Reply:
done